.github/workflows/cpu_ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Install Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.9"
```

Comment:
The latest version of lm_eval does not support Python 3.8.

Reply:
Yep this is fine